netbout-web/src/test/java/com/netbout/rest/MarkdownTest.java

```diff
@@ -39,6 +39,26 @@
 public final class MarkdownTest {
 
     /**
+     * Markdown can handle whitespace after links.
+     * @throws Exception If there is some problem inside
+     */
+    @Test
+    public void handleWithespaceAfterLinks() throws Exception {
+        MatcherAssert.assertThat(
+            new Markdown(
+                "Hi [google](http://www.google.com) how are you?"
+            ).html(),
+            Matchers.allOf(
+                Matchers.equalTo(
+                    // @checkstyle LineLengthCheck (1 line)
+                    "<p>Hi \n<a href=\"http://www.google.com\">google</a> how are you?</p>\n"
```

Comment:
@caarlos0 I agree, but this happens because of JTIDY library. As the project is using XML the library insert a new line before each node. So if you try the following code you will se that before "code" node there is a new line.

    Hi [google](http://www.google.com) how are you?
    
    Line of code: `code` text.
    
    Text.

It wi result on the following code:

    <p>Hi 
    <a href="http://www.google.com">google</a> how are you?</p>
    <p>Line of code: 
    <code>code</code>text.</p>
    <p>text.</p>

That space do not change how user see the message.